src/Infrastructure/Service/LocalCliRunner.php

```diff
@@ -77,15 +77,21 @@ public function canHandle(CommandInterface $command): bool
      */
     public function execute(CommandInterface $command, ContextInterface $context): ContextInterface
     {
-        $placeholders = array_map(function (mixed $placeholder): string {
+        $placeholders = array_map(function ($placeholder): string {
             return '/' . preg_quote((string)$placeholder, '/') . '/';
         }, array_keys($context->getResolvedValues()));
 
-        $values = array_map(function (mixed $value): string {
-            return match (gettype($value)) {
-                'array' => implode(',', $value),
-                default => (string)$value,
-            };
+        $values = array_map(function ($value): string {
+            switch (gettype($value)) {
```

Comment:
Please extract logic to the separate method => `castTostring($value)` and get rid of switch clause by providing early return